### DISCUSSION TITLE
Nuxt config components dirs behavior

### DISCUSSION THREAD

**Discussion by Harkonnen13** [nuxt/nuxt#17960] [Questions] [answered]
Hello. Previously i had config like that, for the namespace reasons:

```
export default defineNuxtConfig({
    components: {
        dirs: [
            '~/components', 
            { path: '~/components/controls', extensions: ['vue'] },
        ],
    },
})
```
and it doesn't work for now:

![image](https://user-images.githubusercontent.com/28978908/152989120-c76b6629-d3bb-4214-9aab-740f45b58336.png)

I think, its related to https://github.com/nuxt/framework/pull/3108, but just to be sure - can i make nested dirs inside components directory?

or should i put them into top level directory like this?

```
export default defineNuxtConfig({
    components: {
        dirs: [
            '~/components', 
            '~/controls', 
        ],
    },
})
```

**Comment by pi0** [comment, marked as answer]
This seems like a regression. Would you please open a bug report?

**Reply by Harkonnen13** [reply]
Thank you for fast reply. https://github.com/nuxt/framework/issues/3129
